src/plugins/mer/mersdkmanager.cpp

```diff
@@ -135,11 +139,13 @@ void MerSdkManager::initialize()
             }
         }
 
-        // If debugger became available
+        // If debugger and/or cmake became available
         for (BuildEngine *const engine : Sdk::buildEngines()) {
             for (const QString &targetName : engine->buildTargetNames()) {
-                for (Kit *const kit : kitsForTarget(targetName))
+                for (Kit *const kit : kitsForTarget(targetName)) {
                     ensureDebuggerIsSet(kit, engine, targetName);
+                    ensureCmakeToolIsSet(kit, engine, targetName);
+                }
```

Comment:
I wonder what makes cmake more similar to debugger than qmake w.r.t the way it is handled here?